docs/adr/0004-ancestry-relationships.md

```diff
@@ -88,7 +88,7 @@ The discussion concluded that in the general case, this type of ancestry should
 
 Management of complex Source<->Source relationships isn't made possible in an effective way by any of the proposals in this ADR, which only really improves the "lightweight copies" use case.
 For the simple case where you want to make a subclip without modification, it probably makes more sense to store a Source (or Flow) ID and a timerange, e.g. inside a MAM.
-Further to [Deletion of Content and IDs](https://github.com/bbc/tams/blob/main/docs/adr/decisions/0004-content-deletion.md) it is assumed such a MAM would also arbitrate deletion requests to avoid removing content needed by a subclip.
+Further to [Deletion of Content and IDs](https://github.com/bbc/tams/blob/main/docs/adr/0004-content-deletion.md) it is assumed such a MAM would also arbitrate deletion requests to avoid removing content needed by a subclip.
```

Comment:
Chatting to @samdbmg . Would you mind changing one of the `0004-...` docs to be `0004a-...`? We'll look at putting together a workflow to verify there are no conflicting ADR numbers in future.

Reply:
Done in https://github.com/bbc/tams/pull/43/commits/72e7bb7baf45546c3c85e69af43cfbc411bb2cba